migrations/versions/5b317c8c141e_removed_flag.py

```diff
@@ -0,0 +1,28 @@
+"""removed flag
```

Comment:
Nice to see that you went through a few migrations. Keep in mind that once you've gotten your "final" table layour squared away that it can be helpful to clear out all th old migrations, recreate the empty database, and get 1 nice, clean migration. Not something we'd want to do after having real data in a production database, but it's nice to start without migration clutter.